src/qz/ui/PopupTray.java

```diff
@@ -219,20 +219,21 @@ public void displayMessage(String caption, String text, Level level) {
      * Attaches the specified TrayIcon to the SystemTray
      * @param trayIcon The TrayIcon to attach
      */
-    private void attachSystemTray(TrayIcon trayIcon) {
-        try {
-            if (getSystemTray() != null) {
-                getSystemTray().add(trayIcon);
-            }
-        } catch (AWTException e) {
-            Logger.getLogger(getClass().getName()).log(Level.WARNING,
-                "Tray menu could not be added: {0}", e.getLocalizedMessage());
-            if (SystemUtilities.isLinux()) {
-                Logger.getLogger(getClass().getName()).log(Level.WARNING,
-                "Ubuntu users:  If the tray icon does not appear try running the following:\n\n" +
-                        "    $ gsettings set com.canonical.Unity.Panel systray-whitelist \"['all']\"\n\n" +
-                        "... then log out and log back in to restart Unity.");
-            }
+    private void attachSystemTray(final TrayIcon trayIcon) {
+        if (getSystemTray() != null) {
+                    try{
```

Comment:
Formatting should be corrected here so the entire blocks doesn't show as newly created.